x-pack/plugins/security_solution/public/resolver/models/process_event.ts

```diff
@@ -78,6 +78,17 @@ export function uniquePidForProcess(passedEvent: ResolverEvent): string {
   }
 }
 
+/**
+ * Returns the pid for the process on the host
+ */
+export function processPid(passedEvent: ResolverEvent): string {
+  if (event.isLegacyEvent(passedEvent)) {
+    return String(passedEvent.endgame.pid);
```

Comment:
`processParentPid` is returning a number, but this is casting the values to a string. Can you make this consistent?

Reply:
https://github.com/elastic/kibana/pull/69098/commits/531070ad3a40510b538c0f6d6ea9f7cd396e12cf